people/soniasingla.toml

```diff
@@ -0,0 +1,6 @@
+name = "Sonia"
+github = "soniasingla"
+email = "soniasingla.1812@gmail.com"
+
+[permissions]
+bors.compiler_builtins.review = false
```

Comment:
Presumably this isn't necessary, as nothing is granting all of these members this permission?

Reply:
Yeah, this is not needed.